common/src/main/java/com/wynntils/core/discord/DiscordManager.java

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.core.discord;
+
+import com.wynntils.antiope.core.DiscordGameSDKCore;
```

Comment:
Is this weird package name due to shadowing? How come we do not see that for other 3rd party libraries?

Reply:
Not sure what you mean?

Reply:
That an external library gets stuck under `com.wynntils`. But I see we have the same when shadowing the Hades lib, so I guess I'll just have to live with it. (If it is possible to make shadowing land in e.g. `com.wynntils.shadow.*` instead of just `com.wynntils`, I'd prefer that, but it is separate from this PR.

Reply:
It is in com.wynntils since we made the package (from a fork)

Reply:
Aha, I see. I missed that part. Then it all makes sense.